build.gradle

```diff
@@ -134,3 +134,4 @@ apply from: file('gradle/ant-compat/forbidden-api-rules-in-sync.gradle')
 apply from: file('gradle/documentation/documentation.gradle')
 apply from: file('gradle/documentation/changes-to-html.gradle')
 apply from: file('gradle/render-javadoc.gradle')
+apply from: file('gradle/validation/documentation-lint.gradle')  // should be placed here; needs to be evaluated after documentation.gradle
```

Comment:
There is already a comment about include ordering above I believe, but fine.

Reply:
With lazy evaluation for 'docroot', the evaluation order does not matter so I moved this line to proper location. (I clearly missed the comment "The order of inclusion of these files shouldn't matter".)